guide/14-deep-learning/how_pspnet_works_and_its_implementation_details_within_arcgis.ipynb

```diff
@@ -0,0 +1,234 @@
+{
```

Comment:
<p>We can see that the receptive field for dilated convolution is larger as compared to the standard convolution. Size of receptive field indicates how much we use context information. In PSPNet the last two blocks of the backbone are dilated with dilation values 2 and 4 respectively.</p><blockquote>the dilated convolution is now calculated on feature maps by multiplying with weights with sparsely at a distance equal to the value of dilation.</blockquote><p>I could not understand this part.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/547/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>